includes/shortcodes.php

```diff
@@ -231,26 +233,23 @@ class="js-give-embed-form-modal-opener"
 									class="in-modal"
 									data-src="%1$s"
 									data-autoScroll="%2$s"
-									style="border: 0; visibility: hidden"></iframe>
+									style="border: 0; visibility: hidden; min-height: %3$spx;"></iframe>
 								<button class="close-btn js-give-embed-form-modal-closer" aria-label="%3$s" data-form-id="%4$s">&times;</button>
 							</div>
 						</div>
 						',
 				$iframe_url,
 				$buttonModeActive ? 0 : $isAutoScroll,
 				__( 'Close modal', 'give' ),
-				$uniqueId
+				$uniqueId,
+				$formStartingHeight
 			);
 		}
 
-		printf(
-			'<div class="give-embed-form-wrapper give-loader-type-img%3$s" id="%2$s">
-				%1$s
-			</div>',
-			$iframe,
-			$uniqueId,
-			$buttonModeActive ? ' is-hide' : ''
-		);
+		$isHidden = $buttonModeActive ? ' is-hide' : '';
+		echo "<div class='give-embed-form-wrapper{$isHidden}' id='{$uniqueId}'>{$iframe}<div class='iframe-loader'>";
```

Comment:
@henryholtgeerts use `printf` only for better code formatting

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/4642/commits/f4ad1a17c4a098d4ee6fbb75beb0d812a0896f6a